apps/meteor/client/components/Message/Attachments/QuoteAttachment.tsx

```diff
@@ -18,6 +18,11 @@ const hover = css`
 			border-inline-start-color: ${colors.n600} !important;
 		}
 	}
+
+	blockquote.rcx-attachment__details {
+		blockquote.rcx-attachment__details {
+			margin-block-end: -12px
+	}
```

Comment:
Why not pass directly to `<Attachment.Details ... />`? Even better, why not move it into `Attachment.Details`? It'd avoid a higher-specificity CSS selector, and it could be more succint:

```
& > & {
  margin-block-end: -12px;
}
```

Reply:
Because not all `<Attachment.Details ... />` will be used in Quote (or a blockquote), if we move to Details all the Details for other Attachments (files, images, pinned messages and etc...) will be impacted, and is possible to include a normal blockquote with Markdown inside `<Attachment.Details ... />`
 
I decided to put at `QuoteAttachment` because this is a component only used for Quote messages and the margin only should be applied at the blockquote within another blockquote.